admin/app/models/spree/admin/form_builder.rb

```
@@ -204,23 +204,43 @@ def spree_radio_button(method, tag_value, options = {})
+      # @option options [Boolean] :crop whether to crop the image
+      # @option options [Boolean] :auto_submit whether to auto-submit the form when the file is uploaded
+      # @option options [Boolean] :can_delete whether to show the delete button
+      # @option options [Boolean] :inline whether to display the uploader inline
+      # @option options [Integer] :height the height of the uploader
+      # @option options [Integer] :width the width of the uploader
+      # @option options [Array] :allowed_file_types the allowed file types, defaults to image types
+      # @return [String] HTML string containing the complete form group with direct file upload field
+      def spree_file_field(method, options = {})
+        @template.content_tag(:div, class: 'form-group') do
+          @template.label(@object_name, method, get_label(method, options)) +
+          @template.render('active_storage/upload_form', form: self, field_name: method, **options) +
```

RuboCop (1.81.7)

[convention] Use 2 (not 0) spaces for indenting an expression spanning multiple lines.

(Layout/MultilineOperationIndentation)